### PR TITLE
Audio manager

### PR DESCRIPTION
**Pull request by Kenster833**
First attempt at AudioManager. Using Unreal Documentation to assist, and telesense guess work.

### CODE REVIEW

**Review by CrazyTobi — CHANGES_REQUESTED** [review]
Looks good to me the only thing I would recommend adding is the info box at the top
/****************************************************************************************
 * Copyright: SteveSloth
 * Name: Your Name Here
 * Script: The name that you called the Script
 * Date: the date you started this script
 * Description: Explain what is in this script and what it is used for
 *              yes you can use multiple lines for more information.
 * TODO: write comments here
 * Known Bugs: anything needs to be fixed
 ****************************************************************************************/

**Review by Jeff-Moreau — CHANGES_REQUESTED** [review]

File: Source/SteveSloth/Public/MyAudioManager.h
Comment (Jeff-Moreau) [inline Source/SteveSloth/Public/MyAudioManager.h:1]: This should also have a list of all the Music and SoundFX used in the game and a way for us to access said sounds

**Review by Jeff-Moreau — CHANGES_REQUESTED** [review]

File: Source/SteveSloth/Public/MyAudioManager.h
Comment (Kenster833) [reply Source/SteveSloth/Public/MyAudioManager.h:1]: I'm not entirely sure what you are asking here. 
Like list out 150 sound files that aren't in the game yet?
Also for accessing I can expose a USoundBase Property in details to place an audio file in. 
Just not fully clear on what you want.

Comment (Jeff-Moreau) [reply Source/SteveSloth/Public/MyAudioManager.h:1]: Try using a Map to store all the sound effects in create the empty map expose it to the editor and we can add to it as we go.. Do one for sound effects and one for music

**Review by CrazyTobi — APPROVED** [review]
Looking good

**Review by Jeff-Moreau — APPROVED** [review]
Looks Good

**Pull request merged**